lib/connect-session-sequelize.js

```diff
@@ -11,7 +11,8 @@ var Op = require('sequelize').Op || {}
 var debug = require('debug')('connect:session-sequelize')
 var defaultOptions = {
   checkExpirationInterval: 15 * 60 * 1000, // The interval at which to cleanup expired sessions.
-  expiration: 24 * 60 * 60 * 1000  // The maximum age (in milliseconds) of a valid session. Used when cookie.expires is not set.
+  expiration: 24 * 60 * 60 * 1000,  // The maximum age (in milliseconds) of a valid session. Used when cookie.expires is not set.
+	disableTouch: false // When true, we will not update the db in the touch function call. Useful when you want more control over db writes.
```

Comment:
something's off with the indentation here

Reply:
Fixed.